x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -146,4 +146,12 @@ sequence
   [process where serial_event_id<3] by unique_pid
   [process where true] by unique_ppid
 '''
-expected_event_ids  = [1, 2, 2, 3]
\ No newline at end of file
+expected_event_ids  = [1, 2, 2, 3]
+
+[[queries]]
+query = '''
+sequence
+  [process where false] by unique_pid
+  [process where true] by unique_ppid
```

Comment:
@costin Question: if we have let's say 4 queries (stages) and one of them (no matter which) is no_match (filter => false) then we cannot match any sequence, correct?

Reply:
Yes, hence the test and rule in this PR.